Makefile

```diff
@@ -637,7 +637,7 @@ install-wasmer:
 	install -Dm755 target/release/wasmer $(DESTDIR)/bin/wasmer
 
 install-capi-headers:
-	for header in lib/c-api/*.{h,hh}; do install -Dm644 "$$header" $(DESTDIR)/include/$$(basename $$header); done
+	shopt -s nullglob; for header in lib/c-api/*.{h,hh}; do install -Dm644 "$$header" $(DESTDIR)/include/$$(basename $$header); done
```

Comment:
This will fail in macos (as `shopt` is not shipped there). Perhaps we can remove the `.hh` option for now

Reply:
It's part of the shell on Linux. Interesting.